src/Notifications/Characters/Mail/Killmail.php

```diff
@@ -67,12 +67,16 @@ public function via($notifiable)
     /**
      * Get the mail representation of the notification.
      *
-     * @param  mixed $notifiable
+     * @param  mixed  $notifiable
      * @return \Illuminate\Notifications\Messages\MailMessage
      */
     public function toMail($notifiable)
     {
 
+        if (! parent::shouldProcessKillmail($notifiable)) {
```

Comment:
this should be done in routing section (`via`) - not the parser (`toX`) - since a notification is already ready to be sent at this stage